src/utils/valid_input_reg_exp.ts

```diff
@@ -0,0 +1,5 @@
+const validInputRegExp = new RegExp('^[0-9]*[.]?[0-9]*$');
```

Comment:
What if we make this RegExp invalidate all numbers that don't have numbers after the point?
- `0.0` is valid
- `.0` is valid
- `0.` is invalid

```suggestion
const validInputRegExp = new RegExp('^[0-9]*([.][0-9]+)?$');
```

Reply:
If `0.` is invalid, the user would not be able to enter decimals

Reply:
Ohh I see...